### DISCUSSION TITLE
Calculation of the total error in parameter estimation

### DISCUSSION THREAD

**Discussion by KisterBa** [Open-Systems-Pharmacology/Forum#1131] [Questions & Problems] [answered]
Dear all, 

I am currently trying to reproduce the total error that is reported after the parameter estimation. For testing, I am using measurements of two compounds at the same time point in blood. I use linear scaling and corresponding weights. If I run the estimation using only compound 1 or 2, I get the expected errors of 0.62 and 0.48. If I use both measurements in the same run, the total error is reported as 0.72. 
So I am wondering how the individual errors are summed up to the reported total error. 
Thank you in advance! 

Best, 
Bastian

**Comment by Yuri05** [comment, marked as answer]
Total error is the square root of squared weighted residuals, where each residual is either `simulated-observed` or `log10(simulated)-log10(observed)` (depending on the scaling settings - s. [Mapping Simulation Outputs to Observed Data‌](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#mapping-simulation-outputs-to-observed-data)).

Weights (which are all = 1 as per default) can be set per observed data set and per observed data point as described in [Mapping Simulation Outputs to Observed Data‌](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#mapping-simulation-outputs-to-observed-data)

For the small values below LLOQ residual calculation is done depending on the LLOQ handling settings as described in [Handling of LLOQ values‌](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#handling-of-lloq-values)

**Reply by KisterBa** [reply]
Thank you a lot! Now I could reproduce the total error!
